Gemfile.lock

```diff
@@ -3,17 +3,17 @@ PATH
   specs:
     ramsey_cop (0.13.2)
```

Comment:
I think you need to bundle again. Otherwise looks great. Thanks for doing this.

Reply:
Good catch. Fixed it.